ci-operator/config/open-cluster-management/application-ui/open-cluster-management-application-ui-main.yaml

```diff
@@ -28,6 +28,42 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: e2e-cluster
+  steps:
+    dependencies:
+      COMPONENT_IMAGE_REF: application-ui
+    env:
+      CLUSTERPOOL_GROUP_NAME: AppLifecycle
+      CLUSTERPOOL_HOST_NAMESPACE: app
+      CLUSTERPOOL_HOST_PROW_KUBE_SECRET: ocm-app-ui-clusterpool
+      CLUSTERPOOL_LIST_INCLUSION_FILTER: prow
+    test:
+    - as: test
+      commands: |
+        export GITHUB_PRIVATE_URL=$(cat "/etc/e2e-secrets/github-private-url") \
+        export GITHUB_USER=$(cat "/etc/e2e-secrets/github-user") \
+        export GITHUB_TOKEN=$(cat "/etc/e2e-secrets/github-token") \
+        export OBJECTSTORE_PRIVATE_URL=$(cat "/etc/e2e-secrets/objectstore-private-url") \
+        export OBJECTSTORE_ACCESS_KEY=$(cat "/etc/e2e-secrets/objectstore-access-key") \
+        export OBJECTSTORE_SECRET_KEY=$(cat "/etc/e2e-secrets/objectstore-secret-key") \
+        export HELM_PRIVATE_URL=$(cat "/etc/e2e-secrets/helm-private-url") \
+        export HELM_USERNAME=$(cat "/etc/e2e-secrets/github-user") \
+        export HELM_PASSWORD=$(cat "/etc/e2e-secrets/github-token") \
+        export HELM_CHART_NAME=$(cat "/etc/e2e-secrets/helm-chart-name") \
+        export ANSIBLE_URL=$(cat "/etc/e2e-secrets/ansible-url") \
+        export ANSIBLE_TOKEN=$(cat "/etc/e2e-secrets/ansible-token") \
+        export SELF="make -f Makefile.prow" \
+        make -f Makefile.prow test-e2e
```

Comment:
I think it would be better to just call the `/opt/app-root/src/application-ui/tests/run-prow-e2e.sh` script here, and let it read all the secrets out of the mount points.

Reply:
@KevinFCormier should it be `./start-cypress-tests.sh` as it's the entry point for the test image?https://github.com/open-cluster-management/application-ui/blob/main/Dockerfile.cypress#L21

Reply:
My thinking was that you would put `run-prow-e2e.sh` inside the image along side `start-cypress-tests.sh`, and invoke the latter as the last command. In `run-prow-e2e.sh` we could read the secrets from `/etc/e2e-secrets` and also set up the other required environment variables as you started working on there. So the main difference would be that instead of calling docker and passing in all the variables, we just call `start-cypress-test.sh` as the final step. (I described this here as well: https://github.com/open-cluster-management/application-ui/pull/1254/files#r643470208) Do you think that will work? Any issues you see?

Reply:
Ah makes sense! No I was confused whether we should run the run-prow-e2e.sh /  start-cypress-test.sh at this stage. Everything is clear now. I will add the step in the Dockerfile.cypress to copy the run-prow-e2e.sh to the same folder as start-cypress-test.sh.